third_party/meshroom/meshroom/nodes/ARTwin/HololensPointcloudComposer.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import print_function
+
+__version__ = "0.1"
+
+from meshroom.core import desc
+import shutil
+import glob
+import os
+import sys
+os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"
+
+# import mapper packages
+dir_path = __file__
+for i in range(6):
+    dir_path = os.path.dirname(dir_path)
+sys.path.append(dir_path)
+from src.holo.HoloIO import HoloIO
+
+class HololensPointcloudComposer(desc.Node):
+
+    category = 'ARTwin'
+    documentation = '''
+This node transform the depthmaps from hololens recording into one coordinate system.
+'''
+
+    inputs = [
+        desc.File(
+            name="recordingDir",
+            label="Recording Folder",
+            description="HoloLens recording directory (images in pv/*.jpg, " \
+                "depth in long_throw_depth/*.pgm, + related .csv).",
+            value="",
+            uid=[0],
+        ),
+        desc.File(
+            name="UVfile",
+            label="UV decoding file",
+            description="Text file with transformation data from depthmap to " \
+                "depth camera coordinte system (uvdata.txt).",
+            value="",
+            uid=[0],
+        ),
+        desc.ChoiceParam(
+            name='verboseLevel',
```

Comment:
There are different types of quotation marks used for strings ' and ". It would be best for consistency reasons to use only one type where it is possible.